prime-router/settings/STLTs/AR/AR-phd.yml

```diff
@@ -0,0 +1,66 @@
+# To submit this to localhost, run:
+#
+# ./prime multiple-settings set --env staging --input ./settings/staging/0170-ar-settings.yml
+---
+- name: "ar-phd"
+  description: "Arkansas Public Health Department"
+  jurisdiction: "STATE"
+  stateCode: "AR"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "ar-phd"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "classpath:/metadata/hl7_mapping/receivers/STLTs/AR/AR-receiver-transform.yml"
```

Comment:
Should we use azure:/ instead of classpath:/?

Reply:
We should. the validate-schemas command was broken but Victor put in a PR to fix it so I can change this back to use Azure.